covid19-dashboard/src/DataTable.tsx

```diff
@@ -212,15 +212,20 @@ export default class DataTable extends React.Component<
         placeFullName = place.name;
       }
 
+      let onClick = () => {};
+      if (subregionType) {
+        onClick = () => goToPlace(place.geoId, subregionType)
+      }
+
       return (
         <tbody key={index}>
-        <tr key={index}
-            className={clickableClass}
-            onClick={() => goToPlace(place.geoId, place.getSubregionType())}>
-          <th>{tableRanking}</th>
-          <th>{placeFullName}</th>
-          {thValues}
-        </tr>
+          <tr key={index}
+              className={clickableClass}
+              onClick={onClick}>
```

Comment:
instead, only register the onclick if there is a subregion. and... add a class if it's clickable so you can give a click affordance to users, e.g changing the cursor and/or underline if it's clickable

Reply:
Please take a look at the last commit, let me know if that's what you meant by "only register the onClick" and "add a class if it's clickable". Thanks!

Sorry, I had to force-push because I made a mistake and reverted.